utils/flink-components-testkit/src/main/scala/pl/touk/nussknacker/engine/flink/util/test/FlinkTestScenarioRunner.scala

```diff
@@ -50,6 +55,7 @@ private object testComponents {
 
 class FlinkTestScenarioRunner(
     val components: List[ComponentDefinition],
+    val globalVariables: Map[String, WithCategories[AnyRef]],
```

Comment:
Please use just a `Map[String, AnyRef]` instead.

Reply:
At the end I have to convert globalComponents to definitions, therefore I use `       GlobalVariableDefinitionExtractor.extractDefinitions(testComponentsHolder.globalVariables)` which requires `Map[String, WithCategories[AnyRef]]`

Reply:
So what's the problem with converting one map to another? Try it and let me know where you find a problem. Remember, that API should be easy to use because many will use it. The implementation can sometimes be more complex

Reply:
Hmm, ok I've done something like this:

```
        GlobalVariableDefinitionExtractor.extractDefinitions(testComponentsHolder.globalVariables.view.mapValues {
          case v: WithCategories[AnyRef] => v
          case v                         => WithCategories.anyCategory(v)
        }.toMap)
```